teams/types.toml

```diff
@@ -0,0 +1,26 @@
+name = "types"
+subteam-of = "compiler"
+
+[people]
+leads = ["nikomatsakis", "jackh726"]
+members = [
+    "compiler-errors",
+    "jackh726",
+    "lcnr",
+    "nikomatsakis",
+    "oli-obk",
+    "spastorino",
+]
+alumni = []
+
+[website]
+name = "Types team"
+description = "Working to implement and formally define the semantics of the Rust language"
+repo = "https://github.com/rust-lang/types-team"
+zulip-stream = "t-types"
+
+[[github]]
+orgs = ["rust-lang", "rust-lang-nursery"]
+
```

Comment:
You can also add a `zulip-group` here to automatically sync this team with a Zulip user group. For example: https://github.com/rust-lang/team/blob/9de1895b14f7515b4bc53a682628de01f8ca0cf6/teams/infra.toml#L51-L52

Reply:
This is a group, not a stream, right?

Reply:
Correct. We don't yet have support for syncing topics.